src/eureka/S3_data_reduction/source_pos.py

```diff
@@ -162,9 +162,12 @@ def source_pos(flux, meta, shdr, m, n, plot=True, guess=None):
         src_ypos, src_ywidth = source_pos_gauss(flux, meta, m, n, plot)
     elif meta.src_pos_type == 'hst':
         src_ypos = guess
-    else:
+    elif meta.src_pos_type == 'max':
         # brightest row for source location
         src_ypos = source_pos_median(flux, meta, m, n, plot)
+    else:
```

Comment:
We should make this an `elif` to do a type check here (e.g. `isnumeric(meta.src_pos_type)`) and have an `else` statement that throws an exception or something